synapse/lib/cell.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, dirn, conf=None):
         roots = self.vault.getRootCerts()
         SessBoss.__init__(self, auth, roots)
 
-        self.cellinfo = {}
+        self.cellinfo = {'type': '%s.%s' % (self.__class__.__module__, self.__class__.__name__)}
```

Comment:
lets use ``ctor`` since it is consistent with other synapse code which describes the class path of a object.